### ISSUE TITLE
Registry not being deployed

### ISSUE THREAD

**Issue by carroarmato0** [kubernetes/minikube#4225]
The cluster has been successfully started with the following command:
```bash
$ minikube start --cpus 4 --memory 8192 --vm-driver kvm2 --insecure-registry '127.0.0.1:11337'
```

The Registry addon has been enabled:
```bash
$ minikube addons enable registry
✅  registry was successfully enabled
```

However, only the service is created, not the pod:
```
$ kubectl get service registry -o wide -n kube-system
NAME       TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE     SELECTOR
registry   ClusterIP   10.111.80.121   <none>        80/TCP    4m57s   kubernetes.io/minikube-addons=registry
$ kubectl get pod registry -o wide -n kube-system
Error from server (NotFound): pods "registry" not found

```

Tested on:
Ubuntu: 19.04
minikube version: v1.0.1
QEMU emulator version 3.1.0 (Debian 1:3.1+dfsg-2ubuntu3)
docker-machine version 0.16.1, build cce350d7

**Comment by carroarmato0** [comment]
No errors while executing the command to enable the registry:

```
[carroarmato0@neon-flower:~] $ minikube addons enable registry -v=10
Found binary path at /home/carroarmato0/.local/bin/docker-machine-driver-kvm2
Launching plugin server for driver kvm2
Plugin server listening at address 127.0.0.1:33283
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(minikube) Calling .GetState
Found binary path at /home/carroarmato0/.local/bin/docker-machine-driver-kvm2
Launching plugin server for driver kvm2
Plugin server listening at address 127.0.0.1:41951
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(minikube) Calling .GetSSHHostname
(minikube) Calling .GetSSHPort
(minikube) Calling .GetSSHKeyPath
(minikube) Calling .GetSSHUsername
Making call to close driver server
(minikube) Calling .Close
Successfully made call to close driver server
Making call to close connection to plugin binary
Making call to close driver server
(minikube) Calling .Close
(minikube) DBG | Closing plugin on server side
Successfully made call to close driver server
Making call to close connection to plugin binary
✅  registry was successfully enabled

```

**Comment by AkarshES** [comment]
This commit fixes the issue: https://github.com/kubernetes/minikube/commit/40da5023d05354e1155942d2aab242411dd10bd7

**Comment by tstromberg** [comment]
Fixed in v1.1.0.
